packages/cli/src/ceramic-daemon.ts

```diff
@@ -41,6 +45,15 @@ const DEFAULT_PORT = 7007
 const HEALTHCHECK_RETRIES = 3
 const CALLER_NAME = 'js-ceramic'
 
+const ADMIN_CODE_EXPIRATION_TIMEOUT = 1000 * 60 * 1 // 1 min
+
+type AdminCode = string
+type Timestamp = number
+
+type AdminCodeCache = {
```

Comment:
I'm worried about this growing unbounded and old entries never being cleaned up.  Can we use an `lru.LRUMap` (see state-cache.ts for an example) with a fixed size (of... 500?) just to make sure that there's some upper bound on how much memory this can consume?